CLAUDE.md

```
@@ -0,0 +1,114 @@
+uv run roborak review --base main  # dogfood on your own branch before asking for review
+uv run python evals/run.py         # 30 labeled cases; real model calls, out of PR CI, run nightly
+```
+
+Docs site (`docs/`, Vite + React Router + Tailwind, Node 20+):
+`cd docs && npm ci && npm run dev | npm run typecheck | npm run build | npm run serve`.
+
+## Architecture
+
+One directional pipeline; each stage only knows the stage before it.
+
+```
```

markdownlint-cli2 (0.23.2)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)